src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/AzureTarget.tsx

```diff
@@ -0,0 +1,312 @@
+            Azure account. The image will be uploaded to the resource group in
+            the subscription you specify.
+          </Text>
+          <Text>
+            To authorize Image Builder to push images to Microsoft Azure, the
+            account owner must configure Image Builder as an authorized
+            application for a specific tenant ID and give it the role of
+            &quot;Contributor&quot; for the resource group you want to upload
+            to. This applies even when defining target by Source selection.
+            <br />
+            <Button
+              component="a"
+              target="_blank"
+              variant="link"
+              icon={<ExternalLinkAltIcon />}
+              iconPosition="right"
+              isInline
+              href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow"
+            >
+              Learn more about OAuth 2.0
+            </Button>
+          </Text>
+        </TextContent>
+        <FormGroup label="Share method:" isRequired>
+          <Radio
+            name="azure-share-source"
+            id="azure-share-source"
+            label="Use an account configured from Sources."
+            description={
+              'Use a configured source to launch environments directly from the console.'
+            }
+            checked={!manual}
+            isChecked={!manual}
+            onClick={() => {
+              setManual(false);
+              clearState();
+            }}
+          />
+          <Radio
+            name="azure-share-manual"
+            id="azure-share-manual"
+            label="Manually enter the account information."
+            checked={manual}
+            isChecked={manual}
+            onClick={() => {
+              setManual(true);
+              clearState();
+            }}
+          />
+        </FormGroup>
+        {!manual && (
+          <>
+            <SourcesSelect
+              provider="azure"
+              selectedSourceId={sourceId}
+              setSelectedSourceId={setSourceId}
+              setFetchingError={setIsErrorFetchingSources}
+            />
+            <SourcesButton />
+            <>
+              {isErrorFetchingSources && (
+                <Alert
+                  variant={'danger'}
+                  isPlain={true}
+                  isInline={true}
+                  title={'Sources unavailable'}
+                >
+                  Sources cannot be reached, try again later or enter an account
+                  info for upload manually.
+                </Alert>
+              )}
+              {(isErrorFetchingSources || isErrorFetchingDetails) && (
+                <Alert
+                  variant={'danger'}
+                  isPlain
+                  isInline
+                  title={'AWS details unavailable'}
+                >
+                  Could not fetch Tenant id and Subscription id from Azure for
```

Comment:
This is the same in the original, but let's update it so it's unified with the names of the fields below.

```suggestion
                  Could not fetch Tenant GUID and Subscription ID from Azure for
```